### PR TITLE
Imp 36 unit integration tests

### PR DESCRIPTION
**Pull request by SKART1**
#36 implemented

### CODE REVIEW

File: test/se/vidstige/jadb/AdbInputStreamFixtureTest.java [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-import se.vidstige.jadb.AdbFilterInputStream;
-import se.vidstige.jadb.Stream;
 
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 
-public class AdbInputStreamFixture {
+public class AdbInputStreamFixtureTest {
```
Comment (vidstige) [inline test/se/vidstige/jadb/AdbInputStreamFixtureTest.java]: This name is not an improvement.

File: test/se/vidstige/jadb/AdbOutputStreamFixtureTest.java [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 import org.junit.Assert;
 import org.junit.Test;
-import se.vidstige.jadb.AdbFilterOutputStream;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
 
-public class AdbOutputStreamFixture {
+public class AdbOutputStreamFixtureTest {
```
Comment (vidstige) [inline test/se/vidstige/jadb/AdbOutputStreamFixtureTest.java]: This name is not an improvement, please use old name

File: test/se/vidstige/jadb/RealDeviceTestCasesIT.java [hunk 3]
```diff
@@ -13,7 +13,7 @@
 import java.io.InputStream;
 import java.util.List;
 
-public class RealDeviceTestCases {
+public class RealDeviceTestCasesIT {
```
Comment (vidstige) [inline test/se/vidstige/jadb/RealDeviceTestCasesIT.java]: This name is not an improvement, please use old name

File: test/se/vidstige/jadb/AdbInputStreamFixtureTest.java (hunk 1 above)
Comment (SKART1) [inline test/se/vidstige/jadb/AdbInputStreamFixtureTest.java]: Only possible solution to separate  `unit` tests from `integration` test is to give them different names....
It is ugly but only possible solution - I have investigated this problem for 2 hours and did not found a solution. 

P.S. Nevertheless we can choose another name pattern, here are [defaults](http://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html)

**Comment by vidstige** [comment]
Let's keep the integration and unit tests separated for now. If you need to test an internal (default visible) function, just either make it public or properly fake it in `FakeAdbServer` & co.

**Pull request closed without merging**